src/main/java/io/cryostat/agent/MainModule.java

```diff
@@ -242,76 +236,14 @@ public static FlightRecorderHelper provideFlightRecorderHelper() {
         return new FlightRecorderHelper();
     }
 
-    @Provides
-    @Singleton
-    public static FileSystem provideFileSystem() {
-        return new FileSystem();
-    }
-
-    @Provides
-    @Singleton
-    @Named(TEMPLATES_PATH)
-    public static Path provideTemplatesTmpPath(FileSystem fs) {
-        try {
-            return fs.createTempDirectory(null);
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
-    }
-
-    @Provides
-    @Singleton
-    public static Environment provideEnvironment(@Named(TEMPLATES_PATH) Path templatesTmp) {
-        return new Environment() {
-            @Override
-            public String getEnv(String key) {
-                if (LocalStorageTemplateService.TEMPLATE_PATH.equals(key)) {
```

Comment:
Is there any more use for `TEMPLATE_PATH`? If not, it can be taken out at line 68